### PR TITLE
Added custom elements in document head

### LINKED ISSUE #18: Document does not have a meta description
Meta descriptions may be included in search results to concisely summarize page content. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/description)

### PR DESCRIPTION
**Pull request by Muljayan**
closes #18 

devtools

> The only draw back of this approach is that crawlers which can't read javascript will not detect the custom metas / titles

>This pr is the same as the one in #43 but made with a different branch.

### CODE REVIEW

File: src/routes/Issues/components/Issues.tsx [hunk 1]
```diff
@@ -109,6 +110,7 @@ export default class Issues extends React.PureComponent<
 
     return (
       <div className="row issues-container">
+        <FixMeMetas title="Fixme | Issues" description="Issues page meta description" />
```
Comment (alexdor) [inline src/routes/Issues/components/Issues.tsx]: The description isn't really descriptive, maybe something like "Open issues waiting for your contribution can be found here" (or something similar) would be more descriptive

File: src/routes/Projects/components/Projects.tsx [hunk 2]
```diff
@@ -27,6 +28,7 @@ export default class Projects extends React.PureComponent<IProjectProps, {}> {
     const { projects } = this.props;
     return (
       <div className="projects-container">
+        <FixMeMetas title="Fixme | Projects" description="projects page meta description"/>
```
Comment (alexdor) [inline src/routes/Projects/components/Projects.tsx]: The description isn't really descriptive, maybe something like "Projects included in the FixMe issue listing" (or something similar) would be clearer

File: src/routes/Home/components/Home.tsx [hunk 3]
```diff
@@ -42,6 +43,7 @@ public onClick=()=> {
     const {projectLength} = this.props
   return (
     <div className="row home-container">
+      <FixMeMetas title="Fixme" description="home page meta description"/>
```
Comment (alexdor) [inline src/routes/Home/components/Home.tsx]: The description isn't really descriptive, maybe something like "FixMe helps you find contribution opportunities across multiple open source projects" (or something similar) would be clearer

**Comment by Muljayan** [comment]
@alexdor will fix them ASAP.

**Comment by Muljayan** [comment]
Hi @alexdor i have included the requested changes and resolved the merge conflicts . Please review it.

**Comment by Muljayan** [comment]
@alexdor  Will be temporary closing this as i noticed it has some conflicts .Will fix it within the day

**Pull request closed without merging**